lib/init.coffee

```diff
@@ -211,10 +211,18 @@ module.exports =
           messages = @parseSchematronMessages(textEditor, output)
         else
           messages = @parseSchemaMessages(textEditor, output)
-        for message in messages
-          message.type = 'Error'
-          message.text = message.text + ' (' + schemaUrl + ')'
-          message.filePath = textEditor.getPath()
+        if messages.length
+          for message in messages
+            message.type = 'Error'
+            message.text = message.text + ' (' + schemaUrl + ')'
+            message.filePath = textEditor.getPath()
+        else if output.indexOf('- validates') is -1
+          messages.push({
+            type: 'Error'
+            text: output
+            filePath: textEditor.getPath()
+            range: [[0, 0], [0, 0]]
```

Comment:
If you omit this range it is still a valid message, linter will show it as associated with the file but not give it a range within the file.